### PR TITLE
Obtain and obtain all

### PR DESCRIPTION
**Pull request by temniipushistik**

### CODE REVIEW

File: TgBot/TourManagerIO/src/main/java/ru/home/tourManagerBot/BotImplementation.java [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 public class BotImplementation extends TelegramLongPollingBot {
   // old private static final String TOKEN = "2067787448:AAEsIUyOxkUGdB4SrRn0aJHprr3IsjzwUOk";
-    private static final String TOKEN = "";
+    private static final String TOKEN = "2067787448:AAGO1pLywCQ6tJHO-wif2aUqQQfqG8c7SxU";
```
Comment (ruykarpuni) [inline TgBot/TourManagerIO/src/main/java/ru/home/tourManagerBot/BotImplementation.java:12]: Remove it

File: TgBot/TourManagerIO/src/main/java/ru/home/tourManagerBot/commands/ObtainAllClients.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        }
+        SendMessage sendMessage = new Start().run(update);
+        sendMessage.setChatId(update.getMessage().getChatId() + "");
+
+        sendMessage.setText(textMessage);
+        return sendMessage;
+    }
+
+
+       /* private void setConfirmationKeyboardMarkup() {
```
Comment (ruykarpuni) [inline TgBot/TourManagerIO/src/main/java/ru/home/tourManagerBot/commands/ObtainAllClients.java:57]: Remove it

**Review by ruykarpuni — CHANGES_REQUESTED** [review]
fix it please

File: TgBot/TourManagerIO/src/main/java/ru/home/tourManagerBot/BotImplementation.java (hunk 1 above)
Comment (temniipushistik) [reply TgBot/TourManagerIO/src/main/java/ru/home/tourManagerBot/BotImplementation.java]: the issue has been resolved

File: TgBot/TourManagerIO/src/main/java/ru/home/tourManagerBot/commands/ObtainAllClients.java (hunk 2 above)
Comment (temniipushistik) [reply TgBot/TourManagerIO/src/main/java/ru/home/tourManagerBot/commands/ObtainAllClients.java]: the issue has been removed

**Pull request merged**